site/en/docs/handbook/how-to/add-a-blog-post/index.md

```diff
@@ -37,6 +37,22 @@ shortcode snippet, it should look like this:
 
 Paste that into the `hero` field in your YAML frontmatter.
 
+### Social sharing image 
+
+Social sharing image is the visual thumbnail that appears when a post or page 
+is shared on social media platforms like Twitter and Facebook.
+
+Follow the [add media guide](/docs/handbook/how-to/add-media/) to upload your
+image to our CDN or use the [sharing image generator](https://web-dev-uploads.web.app/sharing-image-generator)
```

Comment:
I get a 404 at that URL.

Reply:
@rachelandrew  Sorry, Matthias and me have fixed this issue. The sharing image generator URL (https://web-dev-uploads.web.app/sharing-image-generator) is now available. Please re-review it again, thank you ☺️